src/eval.jl

```diff
@@ -161,6 +171,10 @@ function to_code_info(ex, mod, funcname, slots)
     # TODO: Set based on Base.@assume_effects
     purity              = 0x0000
 
+    # TODO: Should we set these?
+    rettype             = Any
+    has_image_globalref = false
```

Comment:
IIUC `rettype` is inferred so we can just set that to `Any`.

However, looking at the code in Base, I believe we need to set `has_image_globalref` when the `code` array contains any `GlobalRef` to a module which `jl_object_in_image` returns true for?

https://github.com/JuliaLang/julia/blob/0e1aa6c7ebb7e328e630ea206e01eb3f64a83d81/src/method.c#L524-L527